engine/classes/Elgg/Database/Annotations.php

```diff
@@ -369,7 +369,7 @@ function getEntities(array $options = array()) {
 			$options['annotation_created_time_lower']);
 	
 		if ($time_wheres) {
-			$options['wheres'] = array_merge($options['wheres'], $time_wheres);
+			$options['wheres'][] = $time_wheres;
```

Comment:
my suggestion was not related to how the data is appended to the array, but was just a suggestion to add `wheres` declared as an empty array to the `$defaults` array, but as @mrclay stated it is not required